nebari-dev/nebari
# Fix hub variable for jupyterhub_dashboard

## Reference Issues or PRs

Fixes JupyterHub Dashboard issue - https://github.com/nebari-dev/nebari/issues/2591



## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ X ] Bug fix (non-breaking change which fixes an issue)

## Testing

- [ X ] Did you test the pull request locally?

## How to test this PR?


Deploy Nebari from this branch.  Make sure at least one JupyterLab user server is running, then check the Grafana dashboard 'Main > JupyterHub Dashboard'.  Several dashboards which were empty prior to this change should now show data.

## Any other comments?
